travis/tests/firstorder/firstorder.out.cmp

```diff
@@ -31,16 +31,16 @@ AtChem2 v1.3-dev
  Finished reading species names.
 
  Reading initial concentrations...
-      1 A                1.000E+02
-      2 B                0.000E+00
+      1 A                                                        1.000E+02
```

Comment:
Not that it matters, but what is going on with all these extra spaces in the `*.out` files?

Reply:
I'd imagine it's the text editor you're using doing some automatic conversion, likely something about how many spaces are equivalent to 1 tab.

Reply:
Must be something like that because I don't think I changed any thing that do that. One of these days I am going to get rid of all tabs in the code :)